apollo-router/src/router_factory.rs

```diff
@@ -795,10 +823,31 @@ mod test {
 
     register_plugin!("test", "always_fails_to_start", AlwaysFailsToStartPlugin);
 
+    async fn create_service(
```

Comment:
moving this above the tests for (at least to me) cleaner organization; down to move it back if that annoys folks